src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -392,6 +411,30 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 		return !!isFullyRendered;
 	}
 
+	private renderTreeData(data: IChatResponseProgressFileTreeData, element: ChatTreeItem, disposables: DisposableStore, templateData: IChatListItemTemplate): { element: HTMLElement; dispose: () => void } {
```

Comment:
FYI I have an issue for keeping some view state for editors when scrolling around the list: https://github.com/microsoft/vscode-copilot/issues/138. That is probably relevant to you too for keeping the expansion state of the tree nodes. I think that's low priority though